doc/classes/EditorFileDialog.xml

```diff
@@ -19,6 +19,15 @@
 				For example, a [param filter] of [code]"*.tscn, *.scn"[/code] and a [param description] of [code]"Scenes"[/code] results in filter text "Scenes (*.tscn, *.scn)".
 			</description>
 		</method>
+		<method name="add_option">
+			<return type="void" />
+			<param index="0" name="name" type="String" />
+			<param index="1" name="values" type="PackedStringArray" />
+			<param index="2" name="index" type="int" />
```

Comment:
The `index` needs clarification (and potentially renaming?) that it's the selected value.

Reply:
Renamed to `default_value_index` and added some comments in the docs.